buildenv/jenkins/JenkinsfileBase

```diff
@@ -406,9 +406,9 @@ def post(output_name) {
 					uploadToArtifactory()
 				}
 			}
-			//for performance test, achive regardless the build result
+			//for performance test, archive regardless the build result
 			//TODO: Need to soft-code the output dir path
-			def benchmark_test_output_dir = 'jvmtest/performance/odm/ilog_wodm881/leftoverResults';
+			def benchmark_test_output_dir = 'jvmtest/perf/odm/ilog_wodm881/leftoverResults';
 			if (fileExists(benchmark_test_output_dir)) {
```

Comment:
If odm is removed this archive for odm specific could also be removed?

Reply:
Ah yes, ideally, we replace that with an archiving for all perf targets that get run.  Depending on how involved that is, I may defer to an upcoming PR (I will have another one prepped for later this week).